docs/src/main/asciidoc/Getting_Started_Guide.adoc

```diff
@@ -64,7 +64,7 @@ Profile |WildFly {wildflyVersion} Full Platform |WildFly {wildflyVersion} Web Pr
 |JSR-345: Enterprise JavaBeans 3.2 |XCMP 2.0Optional |X(Lite) |XCMP
 2.0Not Available |X(Lite)
 
-|JSR-318: Interceptors 1.2 |X |X |X |X
+|JSR-318: Jakarta Interceptors 1.2 |X |X |X |X
```

Comment:
I think the reference to JSR-318 should be removed here

Reply:
+1